modules/phase_field/src/kernels/PFFracBulkRate.C

```diff
@@ -93,33 +98,27 @@ PFFracBulkRate::computeQpOffDiagJacobian(unsigned int jvar)
   Real c = _u[_qp];
   Real gc = _gc_prop[_qp];
 
-  Real x = _l * _betaval[_qp] + 2.0*(1.0-c) * (_G0_pos[_qp]/gc) - c/_l;
+
+  Real x = _l * _betaval[_qp] + 2.0*(1 - c) * (_G0_pos[_qp]/gc) - c/_l;
```

Comment:
undo all changes `1.0` -> `1`. The one is not an integer and I would very much prefer if we explicitly add the decimal `.0` to floating point literals.